Sources/ProjectSpec/ProjectSpec.swift

```diff
@@ -47,7 +50,8 @@ public struct ProjectSpec {
             }
         }
 
-        public init(carthageBuildPath: String? = nil, createIntermediateGroups: Bool = false, bundleIdPrefix: String? = nil, settingPresets: SettingPresets = .all, developmentLanguage: String? = nil) {
+        public init(currentXcodeVersion: String? = nil, carthageBuildPath: String? = nil, createIntermediateGroups: Bool = false, bundleIdPrefix: String? = nil, settingPresets: SettingPresets = .all, developmentLanguage: String? = nil) {
+            self.currentXcodeVersion = currentXcodeVersion ?? Options.defaultXcodeVersion
```

Comment:
Do you think most users of this option will know to use the `XCODE_VERSION_ACTUAL` format (`0910`) vs just specifying `9.1`? Should we have some logic to accept multiple formats

Reply:
I personally think that’d be a matter of documentation, most users shouldn’t even need to set this, having multiple formats could be overdoing it IMHO.

Reply:
I think I agree with @rahul-malik here. If someone defines a string with a single `.` in it we could just join the parts and pad it out to the expected 4 digit number. This can happen in the `XcodeGenKit` through using an extension on `ProjectSpec` that also uses the default fall back as mentioned above.